Vungle/VungleBannerCustomEvent.m

```diff
@@ -133,6 +128,7 @@ - (void)vungleAdDidDisappear
 - (void)vungleAdWasTapped
 {
     MPLogInfo(@"Vungle video banner was tapped");
+    [self.delegate trackClick];
```

Comment:
@YueVungle, discussing currently with @shiv-vungle - based on a comment from Dravya we do not want to call the `trackClick` method here. Unless otherwise noted, I would remove.

Reply:
Hi Clarke, according to Dravya’s comments, if we disable the automatically tracking and remove trackClick call, how MoPub knows user click.

Reply:
Per @shiv-vungle - he had a discussion with Matt Bennion and Dravya earlier today and confirmed that this line should be removed.

Reply:
Got it, remove this call.